extras/manifests/prometheus-ceph-rules.yaml

```diff
@@ -56,13 +69,13 @@ spec:
     - alert: CephMonHighNumberOfLeaderChanges
       annotations:
         description: 'Ceph Monitor "{{ $labels.job }}": instance {{ $labels.instance
-          }} has seen {{ $value }} leader changes recently.'
+          }} has seen {{ $value }} leader changes per minute recently.'
         message: Storage Cluster has seen many leader changes recently.
         severity_level: warning
         storage_type: ceph
       expr: |
-        rate(ceph_mon_num_elections{job="rook-ceph-mgr"}[15m]) > 3
-      for: 30m
+        rate(ceph_mon_num_elections{job="rook-ceph-mgr"}[5m]) * 60 > 0.95
```

Comment:
Same as above

Reply:
May be it was not generated and pushed as part of earlier PR so comes here.